ecosystem/sep-0038.md

```diff
@@ -372,7 +430,7 @@ Servers may deny access to the API if misuse is detected.
 
 #### Transaction Fees
 
-The quote returned includes any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, the delivery method, amount, country code, or other factors.
+The quote returned includes any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, the delivery method, amount, country code, or other factors. Anchors should include the costs of the asset conversion in the `fee` response field.
```

Comment:
I would rewrite this section to make it clear that fees can be collected by adding a margin to the price, and / or by deducting a fixed amount from the funds provided by or delivered to the client.

Reply:
https://github.com/stellar/stellar-protocol/pull/1204/commits/c0ced09246de7ed4ce71438798ceaed16e93c89a